workflows/prognostic_c48_run/sklearn_runfile.py

```diff
@@ -47,11 +43,15 @@
         if comm.rank == 0:
             logger.info(f"diags: {list(diagnostics.keys())}")
 
-        averages = globally_average_2d_diagnostics(
+        averages = runtime.globally_average_2d_diagnostics(
             comm, diagnostics, exclude=loop._states_to_output
         )
+        profiles = runtime.globally_sum_3d_diagnostics(
+            comm, diagnostics, ["specific_humidity_limiter_active"]
+        )
         if comm.rank == 0:
-            log_scalar(time, averages)
+            runtime.log_mapping(time, averages, "statistics")
+            runtime.log_mapping(time, profiles, "profiles")
```

Comment:
I guess the reason I was thinking it would be nice to hide the need for the logger filehandling setup from the runfile is that it might not be immediately obvious based calls to logging here that it's necessary to adjust line 30 to get a file output.  It will still work, but only log to error, which could be confusing.   Up to you!

Reply:
Could add a keyword_argument like `to_file=True` to remove the separation.

Reply:
Just to add, I think it's fine as is since the usage is not sprawling, yet.  Just thinking if this potentially gets used in more than one place.

Reply:
Okay, I think for now I'd rather leave it as it is, since 1) silently doing a logger setup the first time one calls `log_mapping` is also not obvious, and 2) setting up a logger registry feels a bit overkill for the two loggers we have now. Agreed if/when usage expands we should reconsider!